### ISSUE TITLE
[Feature] Simple Error log uploading infrastructure 

### ISSUE THREAD

**Issue by simonsan** [WerWolv/ImHex#1126]
### What feature would you like to see?

Copied from #1111 

There's two parts to this

    The part where ImHex displays an option to upload the current crash information after a crash
    The api endpoint that accepts these crash information and forwards them to e.g a Discord channel

### How will this feature be useful to you and others?

Getting crash reports makes reproducing and thus fixing bugs more straight forward.

### Request Type

- [ ] I can provide a PoC for this feature or am willing to work on it myself and submit a PR

### Additional context?

I would propose just setting up breakpad:
https://chromium.googlesource.com/breakpad/breakpad/

Further reading:
https://hacks.mozilla.org/2022/06/everything-is-broken-shipping-rust-minidump-at-mozilla/
https://jake-shadle.github.io/crash-reporting/#why-not-just-use-breakpad-crashpad

**Comment by iTrooz** [comment]
Closed because this issue was only opened to make us read some info, and we indeed read it, thanks ^^

**Comment by WerWolv** [comment]
Just as a follow up to this, I indeed looked at all these options and even implemented crashpad into ImHex as a test so thanks a lot for the suggestion. It worked fine, however building it was a major pain because it uses autotools which _really_ don't work well with cross-platform builds. 
Ultimately I decided against it because the generated minidumps didn't really provide the information I usually need to debug issues. Instead, I majorly improved the stack traces that are being printed to the log files and now upload anonymized versions of those if the user consents to it.
